components/ScrollUpButton/ScrollUpButton.module.css

```
@@ -0,0 +1,7 @@
+.ScrollUpButton {
+    position: fixed;
+    bottom: 50px;
+    right: 50px;
+    height: 50px;
+    z-Index: 9999999;
```

What do you think about putting 97 here so that the button is below any modal? I think it makes more sense
```suggestion
    z-Index: 97;
```